extensions/subscriptions/locale/en.yml

```diff
@@ -49,7 +52,6 @@ flarum-subscriptions:
 
   # Translations in this namespace are used in emails sent by the forum.
   email:
-
     # These translations are used in emails sent when a post is made in a subscribed discussion
     new_post:
```

Comment:
We need to find a way to switch out the bottom portion of email translations depending on user prefs.

I couldn't find any way to get the user within the notif blueprint, so I'm not sure how I'd change this without having two separate blueprints. @imorland did this in a Blomstra extension and said he can bring over the changes a couple of weeks ago but I forgot to remind him. 🙈